### PR TITLE
Fix RA(A) array NameId keys, querying by NameIds

### PR DESCRIPTION
**Pull request by rjkip**

### CODE REVIEW

File: src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/RaService.php [hunk 1]
```diff
@@ -77,19 +78,19 @@ public function listRas(Institution $institution)
         $credentials = [];
 
         foreach ($ras as $ra) {
-            if (!isset($identities[$ra->nameId])) {
+            if (!isset($identities[(string) $ra->nameId])) {
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/RaService.php]: Illegal object array offsets.

File: src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/RaService.php [hunk 2]
```diff
@@ -66,10 +67,10 @@ public function listRas(Institution $institution)
 
         $nameIds = array_merge(
             array_map(function (Ra $ra) {
-                return $ra->nameId;
+                return (string) $ra->nameId;
             }, $ras),
             array_map(function (Raa $raa) {
-                return $raa->nameId;
+                return (string) $raa->nameId;
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/RaService.php]: `NameId`s were passed to the Identity entity repository, where it actually queries against NameId **strings**.

**Comment by rjkip** [comment]
:bell:  @DRvanR

**Comment by DRvanR** [comment]
:+1:

**Pull request merged**